### DISCUSSION TITLE
Drawer behavior on electron

### DISCUSSION THREAD

**Discussion by viniciusccarvalho** [quasarframework/quasar#11296] [CLI - Electron mode] [answered]
Hi there, I started a new project and configured the template to use `<q-layout view="hHh LpR fFf">` as the settings. 

When running in dev mode, I can see that the toggle of the drawers respect the top bar, but on electron mode the drawer actually overflows the navbar on top.

![Screen Shot 2021-11-11 at 2 58 57 PM](https://user-images.githubusercontent.com/803893/141361088-fd53eb04-7a3c-40ce-bd73-5b3899155249.png)

Was this supposed to happen on electron?

Thank you

**Comment by hawkeye64** [comment]
@viniciusccarvalho We need to see more code on how you implemented it. There are several options you have. Need to see your QLayout and QDrawer implemetations.

**Reply by viniciusccarvalho** [reply]
Sure, here's the full template:

```
<template>
  <q-layout view="hHh LpR fFf">

    <q-header elevated class="bg-primary text-white">
      <q-toolbar>
        <q-btn dense flat round icon="menu" @click="toggleLeftDrawer" />

        <q-toolbar-title>
          
          Romunus - Rom Manager
        </q-toolbar-title>

        <q-btn dense flat round icon="menu" @click="toggleRightDrawer" />
      </q-toolbar>
    </q-header>

    <q-drawer show-if-above v-model="leftDrawerOpen" side="left" bordered > 
      <!-- drawer content -->
    </q-drawer>

    <q-drawer show-if-above v-model="rightDrawerOpen" side="right" bordered >
      <!-- drawer content -->
    </q-drawer>

    <q-page-container>
      <router-view />
    </q-page-container>

  </q-layout>
</template>
```

**Reply by hawkeye64** [reply]
I really can't see a difference. Look at my Electron/Quasar project here: https://github.com/hawkeye64/electron-quasar-file-explorer-v2
Build it and see if it has the same behavior on your platform.

**Reply by viniciusccarvalho** [reply]
Thanks for the link @hawkeye64 , but I can't build the project. I tried to run `quasar dev` on it and it says that I'm not inside a quasar project. Did I miss something obvious here?

**Reply by hawkeye64** [reply]
Did you remember to run`yarn` first?

**Reply by viniciusccarvalho** [reply]
Oh, sorry I'm not familiar with js tooling. Got it running now. So yes your example works as expected, not sure what's going on on my base app. I just bootstrapped it from the quasar layout builder. I still get the drawers overlaying the header. I'll dig into it and see if I can compare with your sample.

**Comment by viniciusccarvalho** [comment, marked as answer]
found the issue `behavior="desktop"` was missing on my code, which is  the one that the generator creates. It's all good now.

**Reply by glatocha** [reply]
Thanks for sharing the answer. Without much knowing I would consider this as an issue. Basically the layout setting is being ignored.
